test-runner-frontend/src/Views/Dashboard.js

```diff
@@ -6,27 +6,24 @@ import Stack from '@mui/material/Stack';
 import GraphQL_Logo from '../Assets/GraphQL_Java_Logo_v2.png';
 import Alert from '@mui/material/Alert';
 import { useEffect, useState } from 'react';
-//import { onSnapshot, collection } from '@firebase/firestore';
-//import db from './firebase';
+import { onSnapshot, collection } from '@firebase/firestore';
+import db from './firebase';
```

Comment:
i think this file needs committing

Reply:
I was thinking about doing that once we have the Firebase credentials with Chris's account.

Reply:
Or should I upload the file with my credentials to test it?

Reply:
@chriswr95 any thoughts on this?

@DiegoManzanarezx i think we should get this working asap as no one will be able to see it working with live data other than you. im happy to use yours but tomorrow morning, lets go over the account to make sure that we set up the tokens correctly and scope it to correct domains etc

Reply:
Sounds perfect! Thanks for that.